src/components/RadioOption.jsx

```diff
@@ -0,0 +1,184 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+
+// TODO: talk to Doug about best way to handle dynamic classes based on props
+
+const StyledRadioOption = styled.div`
+  .button-container {
```

Comment:
All this should be taken care of in the global reset. See if this works without the rule set. It should.